src/tsconfig.json

```diff
@@ -25,7 +25,7 @@
 
     /* Modules */
     "module": "commonjs",                                /* Specify what module code is generated. */
-    "rootDir": "src",                                    /* Specify the root folder within your source files. */
+    // "rootDir": "src",                                    /* Specify the root folder within your source files. */
```

Comment:
Why comment this out?

Reply:
Commenting this out fixed errors I was getting when importing from other directories for the tests